gen/build_deploy/bash/dcos_generate_config.sh.in

```diff
@@ -40,6 +40,11 @@ if [ ! -d genconf/state ]; then
     mkdir -p genconf/state
 fi
 
+DCOS_BOOTSTRAP_CA=dcos-bootstrap-ca
+
+# kill any existing dcos-boostrap-ca processes before running genconf
+kill $(pidof $DCOS_BOOTSTRAP_CA || true) 2>/dev/null || true
```

Comment:
for the record: if there is something running do we care about sending SIGTERM for potential cleanup, or should we SIGKILL?